source/tutorials/manage-minio-tenant.rst

```diff
@@ -12,11 +12,132 @@ Overview
 --------
 
 This page documents several common procedures for managing an existing 
-MinIO Tenant.
+MinIO tenant. You can view the list of tenants at any time from the MinIO 
+plugin for vSphere.
+
+.. _minio-vsphere-view-tenants:
+
+.. image:: /images/vsphere/minio-tenant-list.png
+   :align: center
+   :width: 90%
+   :class: no-scaled-link
+   :alt: MinIO Tenant View
+
+Select the |vcf| cluster and open the :guilabel:`Configure` tab. 
+Navigate to the :guilabel:`MinIO` section and click :guilabel:`Tenants` to 
+view all MinIO tenants deployed in that cluster.
 
 Expand a MinIO Tenant
 ---------------------
 
+Zone expansion increases the total usable storage capacity of the MinIO tenant.
+Each :ref:`zone <minio-vsphere-inside-tenant>` consists of an independent 
+pool of hosts and their locally-attached storage. MinIO presents all 
+zones in the tenant as a single storage resource for applications to access.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+MinIO creates new Tenant pods on available ESXi hosts in the cluster.
+Each ESXi host *must* have sufficient compute, memory, and direct-attached 
+storage to support each pod provisioned as part of the new tenant zone. 
+MinIO requires one available ESXi host for each MinIO server pod deployed 
+as part of the zone.
+
+All zones in a MinIO tenant use the same storage policy. Ensure each ESXi
+host's locally attached drives are included in the storage policy used by the
+MinIO tenant.
+
+1) Open the :guilabel:`Add New Zone` Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+From the vSphere interface, select the cluster in which you want to 
+deploy the MinIO tenant. 
```

Comment:
Was this probably copy pasted? it seems that it is for creation of a tenant.

Reply:
Ah, yes. Good catch, will fix this along with any other missteps you happen to find.

Reply:
that's all I could see, everything else LGTM